gazelle/modules_mapping/def.bzl

```diff
@@ -50,6 +52,11 @@ modules_mapping = rule(
             doc = "A set of regex patterns to match against each calculated module path. By default, exclude the modules starting with underscores.",
             mandatory = False,
         ),
+        "include_stub_packages": attr.bool(
```

Comment:
Other new features get labeled as "experimental" for a couple releases (eg: `experimental_requirement_cycles`). Do we want to do the same here as `experimental_include_stub_packages`?

Given that it's opt-in and that no other _gazelle_ features are experimental, I'm inclined to leave it as-is.

Reply:
I vote to leave it as is